dist/index.html

```diff
@@ -3,10 +3,68 @@
   <head>
     <meta charset="UTF-8">
     <title>Fitlit</title>
+    <link rel="styles" href="src/css/styles.css">
+    <!-- pick a font style here -->
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    <aside class="main-row" id="leftMainRow">
+      <header>Good Morning ${name}</header>
```

Comment:
Technically, we shouldn't be using interpolation tags on HTML, but this is fine I think for now.